removed empty spaces and changed wording (#43)
iXtenda/iVP-gitbook: home/ivp-planning/machines/scale-objects.md

The scaling objects offers a way to alter the size of every object. This is useful to make adjustments of objects that are not [customizable](customizable-machines.md).

Exercise caution when modifying scale to prevent distortions or errors. When changing an objects size via it's [customization](customizable-machines.md) options is possible, doing so should be preferred over changing it's scale.   

{% hint style="info" %}

> Keep in mind that scaling along the Z-axis is possible for some, it may not apply to most objects.

Reviewer: ```suggestion
Keep in mind that scaling along the Z-axis is possible for some, but not most objects at the moment.
```

The scaling objects offers a way to alter the size of every object. This is useful to make adjustments of objects that are not [customizable](customizable-machines.md).

> Exercise caution when using the scaling function to prevent distortions or errors. When opting for the [customization](customizable-machines.md) options of an object, keep in mind that scaling along the Z-axis is possible for some, it may not apply to most objects.

Reviewer: This still does not communicate that using customizations is preferred over scaling.

```suggestion
Exercise caution when modifying scale to prevent distortions or errors. When changing an objects size via it's [customization](customizable-machines.md) options is possible, doing so should be preferred over changing it's scale.
```

The note about being able to scale along the Z-Axis should be a separate info hint and not in this warning. It has nothing to do with customization options.

The scaling objects offers a way to alter the size of every object. This is useful to make adjustments of objects that are not [customizable](customizable-machines.md).

{% hint style="warning" %}
Use the scaling function with care to avoid deformations or glitches. If it is possible to use a [customization](customizable-machines.md) option of the object, always use the option.
{% endhint %}

{% hint style="info" %}

> Use the scaling function with care to avoid deformations or glitches. If it is possible to use a [customization](customizable-machines.md) option of the object, always use the option.

Reviewer: This section is now duplicated and the original info text here is missing. Bring back the original text, but adjust the wording:

* "horizontal axis" should be Z-Axis
* We do support scaling on the Z-Axis for some objects, just not for most
* Remove the part about it currently being in development

The scaling objects offers a way to alter the size of every object. This is useful to make adjustments of objects that are not [customizable](customizable-machines.md).

{% hint style="warning" %}

> Exercise caution when using the scaling function to prevent distortions or errors. When opting for the [customization](customizable-machines.md) options of an object, keep in mind that scaling along the Z-axis is possible for most, yet not all objects

Reviewer: We're loosing the hint to use customization options for scale when possible here. Also scaling along the Z-axis is not possible for most objects, just for some.
